MPF.md

```diff
@@ -1510,6 +1510,100 @@ When custom certificates are in use distributed cleanup needs to know where to f
 
 * Added in CFEngine 3.20.0, 3.18.2
 
+#### PostgreSQL Configuration
```

Comment:
change section heading to "Federated Reporting PostgreSQL Server Configuration" or similar... aka mention Federated Reporting.

Reply:
Really?

I guess that both of the other headings here also explicitly say Federated Reporting, but it feels redundant given the heading levels.

![2022-05-17_13-41-35](https://user-images.githubusercontent.com/202896/168886831-23b34a71-af46-4fca-b758-60d782eeb51e.png)

Reply:
Yeah, it's nested properly.